src/shared/lib/referenceGenomeUtils.tsx

```diff
@@ -21,8 +25,11 @@ export function isMixedReferenceGenome(studies: CancerStudy[]): boolean {
     const isAllStudiesGRCh38 = _.every(studies, study => {
         return isGrch38(study.referenceGenome);
     });
+    const isAllStudiesGRCm38 = _.every(studies, study => {
+        return isGrcm38(study.referenceGenome);
+    });
     // return true if there are mixed studies
-    return !(isAllStudiesGRCh37 || isAllStudiesGRCh38);
+    return !(isAllStudiesGRCh37 || isAllStudiesGRCh38 || isAllStudiesGRCm38);
```

Comment:
~~These condition looks strange to me. I didn't test this, but shouldn't it be:
```(!isAllStudiesGRCh37 || !isAllStudiesGRCh38 || !isAllStudiesGRCm38)```~~

I think its fine